lib/outputCopperShapes.ts

```diff
@@ -0,0 +1,82 @@
+import { Polygon, Box, BooleanOperations } from "@flatten-js/core"
+import { ShapePath } from "lbrnts"
+import type { ConvertContext } from "./ConvertContext"
+import { polygonToShapePathData } from "./polygon-to-shape-path"
+
+/**
+ * Outputs copper shapes for a given layer by unifying net geometries
+ * and converting them to LightBurn ShapePath objects
+ */
+export const outputCopperShapes = ({
```

Comment:
ctx  _must_ be a second parameter- even if you have named parameters by convention

```suggestion
export const createCopperShapesForLayer = ({layer}, ctx)
```